### PR TITLE
Port content from legacy website

### PR DESCRIPTION
**Pull request by jonatas**
# Description

Checking the [run_job](https://docs.timescale.com/api/latest/actions/run_job/) documentation it brings the `retention_policy` data. So, I'm porting the content from the legacy website.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

### CODE REVIEW

File: api/run_job.md [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
+Run a previously registered job in the current session.
+This works for user-defined actions as well as policies.
+Since `run_job` is implemented as stored procedure it cannot be executed
+inside a SELECT query but has to be executed with [CALL](postgres-call).
 
-|Name|Type|Description|
-|---|---|---|
-| `relation` | REGCLASS | Name of the hypertable or continuous aggregate from which to remove the policy |
+<highlight type="tip">
+Any background worker job can be run in foreground when executed with
```
Comment (Loquacity) [inline api/run_job.md:9]:
```suggestion
Any background worker job can be run in the foreground when executed with
```

File: api/run_job.md [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+`run_job`. This can be useful to debug problems when combined with increased
+log level.
```
Comment (Loquacity) [inline api/run_job.md:11]:
```suggestion
`run_job`. You can use this with an increased log level to help debug problems.
```

File: api/run_job.md [hunk 3]
```diff
@@ -1,24 +1,28 @@
...
 
-### Optional Arguments
+#### Required Arguments [](run_job-required-arguments)
 
-|Name|Type|Description|
-|---|---|---|
-| `if_exists` | BOOLEAN |  Set to true to avoid throwing an error if the policy does not exist. Defaults to false.|
+|Name|Description|
+|---|---|
+|`job_id`| (INTEGER)  TimescaleDB background job id |
```
Comment (Loquacity) [inline api/run_job.md:18]:
```suggestion
|`job_id`| (INTEGER)  TimescaleDB background job ID |
```

File: api/run_job.md [hunk 4]
```diff
@@ -1,24 +1,28 @@
...
+#### Sample Usage [](run_job-examples)
 
 ```sql
-SELECT remove_retention_policy('conditions');
+SET client_min_messages TO DEBUG1;
+CALL run_job(1000);
 ```
 
-Removes the existing data retention policy for the `conditions` table.
+Set log level shown to client to `DEBUG1` and run the job with the job id 1000.
```
Comment (Loquacity) [inline api/run_job.md:27]:
```suggestion
Set log level shown to client to `DEBUG1` and run the job with the job ID 1000.
```

**Review by Loquacity — CHANGES_REQUESTED** [review]
Just some small edits.

**Comment by jonatas** [comment]
Thanks for the suggestions @Loquacity!

**Comment by jonatas** [comment]
@Loquacity I enabled auto-merge for this PR after addressing the changes. Thank you for the suggestions! I'm learning from them!

**Review by Loquacity — APPROVED** [review]

**Pull request merged**